Apps/Examples/Examples/All Examples/CameraAnimatorsExample.swift

```diff
@@ -0,0 +1,79 @@
+import UIKit
+import MapboxMaps
+
+@objc(CameraAnimatorsExample)
+
+public class CameraAnimatorsExample: UIViewController, ExampleProtocol {
+
+    internal var mapView: MapView!
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+
+        guard let accessToken = AccountManager.shared.accessToken else {
+            fatalError("Access token not set")
+        }
+
+        let resourceOptions = ResourceOptions(accessToken: accessToken)
+        mapView = MapView(with: view.bounds, resourceOptions: resourceOptions)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        view.addSubview(mapView)
+
+        mapView.cameraView.centerCoordinate = CLLocationCoordinate2D(latitude: 37.7749, longitude: -122.4194)
+        mapView.cameraView.zoom = 5
+
+        // Allows the delegate to receive information about map events.
+        mapView.on(.mapLoaded) { [weak self] _ in
+
+            guard let self = self else { return }
+
+            // The following dispatch group is used to coordinate the completion of
+            // this example. This is useful in the case where the durations and delays
+            // are changed.
+            let group = DispatchGroup()
+            group.enter()
+            group.enter()
+            group.enter()
+
+            // Define the animation options that will be passed in for each animator.
+            let options : UIView.AnimationOptions = [.beginFromCurrentState, .allowUserInteraction, .curveLinear]
+
+            // Update the map camera's bearing.
+            _ = self.mapView.cameraManager.makeRunningCameraAnimator(duration: 4,
+                                                                     delay: 2,
+                                                                     options: options,
+                                                                     animationOwner: .unspecified) { [self] in
+                self.mapView.cameraView.bearing = 180.0
+            } completion: { (position) in
+                print("done!")
+                group.leave()
+            }
+
+            // Center the map camera over New York City.
+            _ = self.mapView.cameraManager.makeRunningCameraAnimator(duration: 4,
```

Comment:
what about an example using `makeCameraAnimators` as opposed to `makeRunningCameraAnimator`